### DISCUSSION TITLE
How to select "headlight_mode" from an automation or service?

### DISCUSSION THREAD

**Discussion by rhoddan** [Thomas55555/husqvarna_automower#501] [Q&A] [answered]
First - thanks for a great integration!

I can change headlight_mode for my 415x from settings or via the dashboard. How can I do the same from an automation?

**Comment by prairiesnpr** [comment, marked as answer]
Should be able to complete this easily through the automation editor. For the action, choose device, then select the mower. 

![SmartSelect_20230627_082319_Home Assistant](https://github.com/Thomas55555/husqvarna_automower/assets/18637008/2b17e517-e693-44bf-818a-39171fffe5c3)

**Reply by rhoddan** [reply]
Ahh it was easier than I thought. I searched for a specific service. Thanks a lot!
